lib/Runtime/Library/JavascriptError.h

```diff
@@ -84,6 +84,9 @@ namespace Js
         static void __declspec(noreturn) SetMessageAndThrowError(ScriptContext* scriptContext, JavascriptError *pError, int32 hCode, EXCEPINFO* pei);
         static JavascriptError* MapError(ScriptContext* scriptContext, ErrorTypeEnum errorType);
 
+        //HELPERCALL needs a non-overloaded function pointer
+        static void __declspec(noreturn) ThrowUnreachable(ScriptContext* scriptContext, int32 hCode) { ThrowError(scriptContext, hCode); }
```

Comment:
this is a really good idea. I can get rid of that pesky error code constant in Lowerer :-)